src/measure_theory/measure/haar.lean

```diff
@@ -538,38 +530,57 @@ begin
 end
 
 @[to_additive]
-lemma haar_measure_self [locally_compact_space G] {K₀ : positive_compacts G} :
+lemma haar_measure_self {K₀ : positive_compacts G} :
   haar_measure K₀ K₀.1 = 1 :=
```

Comment:
All the proofs using `positive_compacts` would be easier to follow if those `K.1` or `K.2.1` had better names like `K.set`, `K.is_compact`. No need to change that in this PR though.

Reply:
Also why is there no coercion from `positive_compacts` to `set` ?

Reply:
I don't have answers to this question. My guess is that @fpvandoorn went for the quick and dirty way when implementing the Haar measure, but using structures instead of subtypes, with nice names and coercions, would probably be cleaner. I won't mix such a refactor with this PR, though, because it would be more confusing.